packages/compat/src/v1-app.ts

```diff
@@ -243,32 +241,6 @@ export default class V1App implements V1Package {
     return 7;
   }
 
-  private debugMacrosPlugin() {
-    let DebugMacros = require.resolve('babel-plugin-debug-macros');
-    let isProduction = process.env.EMBER_ENV === 'production';
-    let isDebug = !isProduction;
-    let options = {
-      flags: [
-        {
-          source: '@glimmer/env',
-          flags: { DEBUG: isDebug, CI: !!process.env.CI },
-        },
-      ],
-
-      externalizeHelpers: {
-        global: 'Ember',
-      },
-
-      debugTools: {
-        isDebug,
-        source: '@ember/debug',
-        assertPredicateIndex: 1,
-      },
-    };
-    return [DebugMacros, options];
-  }
-
-  @Memoize()
```

Comment:
Was removing `@Memoize` here intentional? This change seems unrelated to changes to the babel plugin config...

Reply:
good catch!

Reply:
fixed